docs/building.md

```diff
@@ -12,29 +34,23 @@ for proper setup. If done correctly, you should have a GOROOT and GOPATH set in
 environment.
 
 If a Go development environment is setup, one can use `go get` to install the
-`registry` command from the current latest:
+`registry` source code from the current latest:
```

Comment:
This `go get` command will actually build registry and install it in $GOPATH/bin. The instructions on this happening, since a few lines down we say to run `$GOPATH/bin/registry`.